configure.ac

```diff
@@ -533,6 +533,9 @@ AC_CHECK_DECLS([bswap_16, bswap_32, bswap_64],,,
                  #include <byteswap.h>
                  #endif])
 
+AC_MSG_CHECKING(for __builtin_clzl)
```

Comment:
Hmm...is it definitely the case that all the compilers we support have this? Can we not have some fallback for those that do not?